media-session/index.js

```diff
@@ -0,0 +1,119 @@
+let audio = document.createElement('audio');
+
+let playlist = getPlaylist();
+let index = 0;
+
+function onPlayButtonClick(event) {
+  playAudio();
+}
+
+function playAudio() {
+  audio.src = playlist[index].src;
+  audio.play()
+  .then(_ => updateMetadata())
+  .catch(error => log(error));
+}
+
+function updateMetadata() {
+  let track = playlist[index];
+
+  log('Playing "' + track.title + '" track...');
+  navigator.mediaSession.metadata = new MediaMetadata({
+    title: track.title,
+    artist: track.artist,
+    album: track.album,
+    artwork: track.artwork
+  });
+}
+
+/* Previous Track & Next Track */
+
+navigator.mediaSession.setActionHandler('previoustrack', function() {
+  log('> User clicked "Previous Track" icon.');
+  index = (index - 1 + playlist.length) % playlist.length;
+  playAudio();
+});
+
+navigator.mediaSession.setActionHandler('nexttrack', function() {
+  log('> User clicked "Next Track" icon.');
+  index = (index + 1) % playlist.length;
+  playAudio();
+});
+
+/* Seek Backward & Seek Forward */
+
+let skipTime= 10; /* Time to skip in seconds */
+
+navigator.mediaSession.setActionHandler('seekbackward', function() {
+  log('> User clicked "Seek Backward" icon.');
+  audio.currentTime = Math.max(audio.currentTime - skipTime, 0);
+});
+
+navigator.mediaSession.setActionHandler('seekforward', function() {
+  log('> User clicked "Seek Forward" icon.');
+  audio.currentTime = Math.min(audio.currentTime + skipTime, audio.duration);
+});
+
+/* Utils */
+
+function getPlaylist() {
+  return [{
+      src: 'jam.ogg',
```

Comment:
Ah... I see.

I have one short recording I did of me playing guitar: https://simpl.info/audio/audio/audio.mp3.  Of course, you'll still need to add a licence. (I probably should add something to simpl.info, even though it's my recording.) I guess I could do four more if need be (if you can face more of my guitar playing :).

Alternatively, I just found these Sintel recordings (used by Dale Curtis for his MSE gapless playback article: https://developers.google.com/web/updates/2015/06/Media-Source-Extensions-for-Audio):

https://github.com/samdutton/simpl/tree/gh-pages/mse/audio/audio